File: easy/412_Fizz_Buzz.py
def fizzBuzz(n: int) -> list[str]:
    new = []
    for i in range(1, n + 1):
        if i % 3 == 0 and i % 5 == 0:
            new.append("FizzBuzz")
        elif i % 5 == 0:
            new.append("Buzz")
        elif i % 3 == 0:
            new.append("Fizz")
        else:
            s = str(i)
            new.append(s)
    return new

File: easy/412_Fizz_Buzz.c
#include <stdlib.h>
#include <stdio.h>

char** fizzBuzz(int n, int* returnSize) {
    char **answer = malloc(sizeof(char *) * n);

    *returnSize = n;

    for (int i = 1; i <= n; i++) {
        if (i % 15 == 0) {
            answer[i - 1] = "FizzBuzz";
        }
        else if (i % 3 == 0) {
            answer[i - 1] = "Fizz";
        }
        else if (i % 5 == 0) {
            answer[i - 1] = "Buzz";
        }
        else {
            answer[i - 1] = malloc(12);
            sprintf(answer[i - 1], "%d", i);
        }
    }
    return answer;
}
